theme/templates/pages/press.html

```diff
@@ -46,19 +46,91 @@ <h2><a class="headline-link" href="#pressmirror">{% trans %}Press Mirror{% endtr
   <div class="grid-container halves">
 
     <div>
-      {% trans %}2019-07-07:{% endtrans %}<br/>
-      <a href="https://djworx.com/mixxx-2-3-beta-brings-colour-and-more/">
-        <h6 class="docs-header">(EN) DJWORX: Mixxx 2.3 Beta Brings Colour And More</h6>
+     {% trans %}2025-08-15:{% endtrans %}<br/>
+      <a href="https://www.deejayplaza.com/en/articles/mixxx-review">
+        <h6 class="docs-header">(EN) DeeJayPlaza: Mixxx Review</h6>
       </a>
-      <p class="pressquote"> "it has features to compete with the big guns of the commercial DJ software sector" </p>
+      <p class="pressquote">"I was pleasantly surprised by how complete and usable it is."</p>
+    </div>
+
+    <div>
+      {% trans %}2025-06-16:{% endtrans %}<br/>
+      <a href="https://www.digitaldjtips.com/reviews/mixxx-2-5-1-dj-software/">
+        <h6 class="docs-header">(EN) Digital DJ Tips: Mixxx 2.5.1 DJ Software Review</h6>
+      </a>
+      <p class="pressquote">"But what you get in return is software that respects you as a DJ, doesn’t try to upsell you, and focuses on the fundamentals."</p>
+    </div>
+
+    <div>
+      {% trans %}2025-03-19:{% endtrans %}<br/>
+      <a href="https://music-worx.com/mixxx-2-5-review-the-best-free-open-source-dj-software/">
+        <h6 class="docs-header">(EN) Music-Worx: Mixxx 2.5 Review: The Best Free Open-Source DJ Software</h6>
+      </a>
+      <p class="pressquote">"Mixxx proves that open-source software can stand toe-to-toe with the industry’s biggest names."</p>
+    </div>
+
+    <div>
+      {% trans %}2025-02-12:{% endtrans %}<br/>
+      <a href="https://www.gearnews.com/mixxx-is-the-best-dj-software/">
+        <h6 class="docs-header">(EN) Gearnews: Mixxx Is the Best DJ Software You're Not Using</h6>
```

Comment:
This is just another reformating of our release announcement. No journalistic work done here.